Makefile

```diff
@@ -19,3 +19,8 @@ build: format vendor
 
 it: build
 	- sh integration_test.sh $(ROOT)
+
+
+querybuilder-example:
```

Comment:
It's better to be in a documentation file and not in MakeFile. Please remove it.